src/components/DocsetMenu/DocsetButton.js

```diff
@@ -1,23 +1,37 @@
 import PropTypes from 'prop-types';
 import React, {useContext} from 'react';
 import path from 'path';
-import {Button} from '@chakra-ui/react';
+import {Button, Link, Tooltip} from '@chakra-ui/react';
 import {Link as GatsbyLink} from 'gatsby';
 import {PathContext} from '../../utils';
 
-export default function DocsetButton({to, ...props}) {
+export default function DocsetButton({to, tooltipLabel, ...props}) {
   const {basePath} = useContext(PathContext);
   const [docsetPath] = basePath.split(path.sep);
+
+  const linkProps = to?.startsWith('/')
+    ? {
+        as: GatsbyLink,
+        to
+      }
+    : {
+        as: Link,
+        href: to,
+        isExternal: true
```

Comment:
Can we make this just use a regular `a` tag so that we don't get the underline effect on hover?

```suggestion
        as: 'a',
        href: to,
        target: '_blank'
```